docs/tutorials/build-an-nft-marketplace.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Build an NFT marketplace
+lastUpdated: 2nd November 2023
+---
+
+This tutorial guides you through creating a web application that allows users to buy and sell NFTs of different types.
+You will use the Taqueria platform to manage smart contracts and a distributed web application (dApp) to handle the backend and frontend of the project.
+
+You will learn:
+
+- What kinds of tokens Tezos supports
+- What token standards are
+- How to create contracts that are based on existing templates for token standards
+- How to store token metadata in distributed storage with IPFS
+- How to handle token transfers and other operations
+- How to list tokens for sale and accept payments from buyers
+
+## Prerequisites
+
+1. Optional: If you haven't worked with Tezos NFTs before, consider doing the tutorial [Create an NFT](./create-an-nft) first.
+
+1. Set up an account with Pinata if you don't have one already and get an API key and API secret.
+For instructions, see the [Configure IPFS storage](./create-an-nft/nft-taquito#configure-ipfs-storage) section of the tutorial [Create a contract and web app that mints NFTs](./create-an-nft/nft-taquito).
+
+1. Make sure that you have installed these tools:
+
+   - [npm](https://nodejs.org/en/download/): NPM is required to install the web application's dependencies
+   - [taqueria >= v0.40.0](https://github.com/ecadlabs/taqueria): Taqueria is a platform that manages the contracts and front ends for Tezos dApps
```

Comment:
Shall we include this link for the taqueria documentation instead of the GitHub link?
https://taqueria.io/

Reply:
Got it, thx!